pom.xml

```diff
@@ -148,33 +148,4 @@
         </dependencies>
     </dependencyManagement>
 
-    <distributionManagement>
```

Comment:
I believe this is required for publishing, without `<distributionManagement>` publishing wouldn't work but not so sure about `<repositories>`

Reply:
That's fine, so would you be willing to confirm that and, if true, repair it?  I'm not sure if there's anything I can do to fix it.  But right now, it doesn't work for fresh installs unless you remove it.  You can add it back once the source is in your .m2 repository but that seems like a very ugly workaround.